cmd/cloudImageRename.go

```diff
@@ -18,17 +18,16 @@ package cmd
 import (
 	"fmt"
 
+	apiTypes "github.com/liquidweb/liquidweb-cli/types/api"
 	"github.com/spf13/cobra"
-
-	"github.com/liquidweb/liquidweb-cli/types/api"
```

Comment:
in the other files the internal package imports have been below non-project imports like `spf13/viper` etc